crates/bevy_ecs/src/query/access.rs

```diff
@@ -59,6 +59,8 @@ pub struct Access<T: SparseSetIndex> {
     /// Is `true` if this has mutable access to all elements in the collection.
     /// If this is true, then `reads_all` must also be true.
     writes_all: bool,
+    // Elements that are not accessed, but whose presence in an archetype affect query results.
+    archetypal: FixedBitSet,
```

Comment:
Shouldn't these be populated by the appropriate QueryData/QueryFilter/SystemParam implementors? or is the plan to handle that in a separate PR?

Reply:
Oh right, forgot to move that over from my branch.